README.md

```diff
@@ -157,7 +157,7 @@ Usage:
 
 ```lua
 local state = jit.prngstate()
-local oldstate = jit.prngstate(123456)
+local oldstate = jit.prngstate({123456})
```

Comment:
This doesn't look right. It requires "an array of 8 Lua numbers", not just one.

Reply:
According to luajit's test case, if the parameter array is shorter than 8, the rest is considered 0.
The use case here looks like typo, missing the curly braces.

Reply:
The code example in the docs should never rely on undocumented features. It's weird.